### ISSUE TITLE
Make "Skip to Break" and "Skip to Work" Functions

### ISSUE THREAD

**Issue by arulster17** [saumyasinghal747/Pomodoro#1]
fjhgredjfogiyj5nedjui8vbhiytkjrbegsyxc8gitk54j3hwqasyudifogly,kmj4euws9dfoghktrnehwgatrts6c7v8gihyktjhrewtsyd8f9igjthnrejiw9sd7c6tfgehjk3l4toy9h8gufdhswjekrtp0g9fiudhwn34besyudiforkmn3bwsyd7yetgfd3ghuertfdwvbnkeortiugfvycuisodikjcfrm3,wlsdiugtrjekrfgifodsieuyrhjdksowirdso9w8ushbidfuhyew6asrzxdfcghgjktyl,reskdfigohp;uik'l[mhpgotyuli[]j[hpgofieuwysdufitklrepwsaz0a9htiejdforpoidrieol4kmrtjure9okrthfuidekrtgfodiurhgthjkfiudhfujkblg;oefduhfjkltoriuwedghsjbfje8wefydvhcdhgte72efywqhvbjswdfogtfeijhurteywhijdoklmkweif89troelkdnfhyuijhgty789oihgfreswxcgu876resxcvghurdsxvbjkigdyvhcjzjd9gj0fuoeifoihsufusfgysebfjejrogjirejgoejgejogjw9gjowjg8hoj8heofjehfioehy6t78wuegwyutr6e56tgujwjfgygshj9oybkwyyfhvgafyagvagdabgaufuwiry7wuekiwuyruwihfuywgeuhugfwbnfkljhuejdfogt9eidjfnbrsvdfghnytrmekdsfjbgnrewsiddufhrejm345tyhobvcdo

TL;DR: ofivhleirsfhguiertieoirhtugrfkedsif8vghujtrekosdiufyghtrjekidhfen3endhfgyudiskrjnbtreius7ydtfgdehwjkaisu7tgwh2neir
